app/templates/components/users-cards.hbs

```diff
@@ -0,0 +1,2 @@
+  {{#user-count}}
```

Comment:
Use angular brackets here

Reply:
I have added angular bracket for this component